### PR TITLE
Ticket - Transaction Detail View

### PR DESCRIPTION
**Pull request by ChristianMoschke**
Pull request for Issue: [Transaction Detail View](https://github.com/miToe/dagobert/issues/8)

In `/pages/index.js` line 18 has been modified to display the date based on user (browser/system) settings.

### CODE REVIEW

**Review by Jonstar22 — COMMENTED** [review]
It looks good to me.

 However, consider naming the page TransactionDetails for better clarity.

**Comment by doemser** [comment]
You have a page that only returns one component, you can make that component the page

**Comment by ChristianMoschke** [comment]
> It looks good to me.
> 
> However, consider naming the page TransactionDetails for better clarity.

The folder structure has been modified `/components/Transaction.js` ->  `/components/Transaction/index.js`

**Comment by Jonstar22** [comment]
looks good now :)

File: src/pages/transactions/[id].js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import Transaction from '../../components/Transaction';
+
+export default function TransactionDetails() {
+  return <Transaction />;
+}
```
Comment (doemser) [inline src/pages/transactions/[id].js:5]: Please make the Transaction component the page and remove unncessary layer

**Review by doemser — CHANGES_REQUESTED** [review]

File: src/pages/transactions/[id].js (hunk 1 above)
Comment (ChristianMoschke) [reply src/pages/transactions/[id].js]: Done

**Review by evabergold — APPROVED** [review]
looks good :-)

**Pull request merged**